subprojects/core-api/src/main/java/org/gradle/api/reflect/TypeOf.java

```diff
@@ -40,7 +39,6 @@
  * @param <T> Parameterized type
  * @since 3.5
  */
-@Incubating
```

Comment:
Shouldn't `getConcreteClass()` still be incubating?

Reply:
Incubated!